frontend/components/sections/BannersSection/SingleBanner.tsx

```diff
@@ -69,6 +69,8 @@ export function SingleBanner({ id, banner }: SingleBannerProps) {
                   >
                      {banner.callToAction.title}
                   </SmartLink>
+               ) : (
+                  <Box h="10" mt="10" />
```

Comment:
This looks a bit a hacky solution... I think it should be possible to convert the container to flex and vertically center that way? 🤔

Reply:
I agree it's a bit hacky/unmaintainable. It's tricky due to the use of padding on the outermost container to determine the height of the image.

I did consider making the container a flex like you suggested. However, when the CTA is omitted the height of the image is reduced. We could conditionalize the padding on the presence of the CTA, but that is just as unmaintainable as this Box solution.

I will spend a little more time with this today to see if there is a cleaner solution. dev_block ✌🏻